contracts/dapis/DapiServer.sol

```diff
@@ -365,6 +378,10 @@ contract DapiServer is
                 this.fulfillPspBeaconUpdate.selector
             )
         );
+        require(
```

Comment:
The only functional change in this PR

Reply:
Kind of related to this. How does one un-registers a subscription? Would that even be allowed? Should we have a `resetBeaconSubscription()` function that resets the entry in the `subscriptionIdToHash` mapping?

Reply:
The registration only has the DapiServer know what Beacon ID a subscription ID maps to. There is no downside to having it remain registered after the subscription stops being used. In contrast, unregistering an active subscription will prevent it from updating the Beacon from then on so I'm not sure who can be feasible allowed to do that, probably `airnode`, maybe `sponsor` but even that is iffy imo.